SConstruct

```diff
@@ -378,6 +380,34 @@ elif env['COMPILER'] == 'msvc':
         env.Append(CCFLAGS=Split('/Od /Zi /MD'))
         env.Append(LINKFLAGS=Split('/DEBUG'))
 
+if not env['SHOW_CMDS']:
+    # Hide long compile lines from the user
+    arch = env['MACOS_ARCH'] if IS_DARWIN else 'x86_64'
```

Comment:
should that be `env['ARCH']` instead of x86_64 ?

Reply:
Nah, arnold-usd doesn't have an ARCH variable, just a MACOS_ARCH variable for Darwin (although it is driven by the ARCH variable from HtoA)